src/lib/components/composites/search-bar/SearchBar.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+    import { Input } from "$lib/components/primitives/input/index";
+    import { Search } from "lucide-svelte";
+
+    interface Props {
+        placeholderText?: string;
+        onSearch: (searchText: string) => void;
+    }
+    const { placeholderText = "Search", onSearch }: Props = $props();
+
+    let searchInput: string = $state("");
+
+    // timeout in ms to wait after last key input before start search
+    const timeout: number = 500;
```

Comment:
If the unit is not inferred by type, I would add it in the name.
```suggestion
    const timeoutInMs: number = 500;
```

Reply:
Agree, I will change it (and keep it in mind for similar cases).

Reply:
This change isn't applied yet.

Reply:
Sorry, accidentally restored this change.